setup.py

```diff
@@ -4,10 +4,16 @@
 install_requires = [
     'sockjs-tornado==1.0.0',
     'tornado==3.2.2',
-    'wsgiref==0.1.2',
-    'argparse'
+    # 'wsgiref==0.1.2',
+    'argparse',
+    'simplejson' # simplejson is really required for py3 support to avoid encoding problems
 ]
 
+try:
+    import wsgiref
+except ImportError:
+    install_requires.append('wsgiref==0.1.2')
```

Comment:
Yeah, I understand. My point is that the wsgiref package is not needed
anymore for thunderpush to work or am I mistaken?

torsdag 8 oktober 2015 skrev uthunderbird <notifications@github.com>:

> In setup.py
> <https://github.com/thunderpush/thunderpush/pull/43#discussion_r41495156>:
>
> >  ]
> >
> > +try:
> > +    import wsgiref
> > +except ImportError:
> > +    install_requires.append('wsgiref==0.1.2')
>
> It's required for python 2.6 and for python 3.1.x as I understand.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thunderpush/thunderpush/pull/43/files#r41495156>.
>